lua/autorun/client/cl_contact_forms.lua

```diff
@@ -810,4 +810,11 @@ net.Receive( "CFC_ContactForms_FailureAlert", function()
     )
 end )
 
+net.Receive("Admin_alert", function()
+    local ply = net.ReadString() -- Reads the reporter's name
+    local plyRankColor = net.ReadColor() -- Reads the color of reporter's rank
+    chat.AddText( plyRankColor, ply, YELLOW, " has submitted a report." ) -- messgae via chat
```

Comment:
Same as mentioned on server side, might be easier to just network the submitter and then get their name and rank color.

Reply:
On client, changed it to read the entity and then set variables to respective values such as name and team color.